src/setup_payload/python/generate_setup_payload.py

```diff
@@ -17,11 +17,15 @@
 #
 import argparse
 import enum
+import os
 import sys
 
 import Base38
 from bitarray import bitarray
-from stdnum.verhoeff import calc_check_digit
+
+CHIP_TOPDIR = os.path.dirname(os.path.realpath(__file__))[:-len(os.path.join('src', 'setup_payload', 'python'))]
```

Comment:
maybe this is more generic:

```
os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', '..')
```

you also remove `src` to add it back, so maybe just go 2 levels up and add the other 2 items:

```py
sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', 'lib', 'support', 'verhoeff'))
```